packages/terra-abstract-modal/src/_ModalContent.jsx

```diff
@@ -70,90 +59,67 @@ const defaultProps = {
   classNameModal: null,
   classNameOverlay: null,
   closeOnOutsideClick: true,
-  fallbackFocus: undefined,
-  isFocused: true,
   isFullscreen: false,
   isScrollable: false,
   role: 'dialog',
   zIndex: '6000',
 };
 
-/* eslint-disable react/prefer-stateless-function */
-class ModalContent extends React.Component {
-  render() {
-    const {
-      ariaLabel,
-      children,
-      classNameModal,
-      classNameOverlay,
-      closeOnOutsideClick,
-      onRequestClose,
-      fallbackFocus,
-      role,
-      isFocused,
-      isFullscreen,
-      isScrollable,
-      zIndex,
-      ...customProps
-    } = this.props;
+const ModalContent = (props) => {
+  const {
+    ariaLabel,
+    children,
+    classNameModal,
+    classNameOverlay,
+    closeOnOutsideClick,
+    onRequestClose,
+    role,
+    refCallback,
+    isFullscreen,
+    isScrollable,
+    zIndex,
+    ...customProps
+  } = props;
+  let zIndexLayer = '6000';
 
-    let zIndexLayer = '6000';
-    if (zIndexes.indexOf(zIndex) >= 0) {
-      zIndexLayer = zIndex;
-    }
-
-    const modalClassName = cx([
-      'abstract-modal',
-      { 'is-fullscreen': isFullscreen },
-      `layer-${zIndexLayer}`,
-      classNameModal,
-    ]);
-
-    // Delete the closePortal prop that comes from react-portal.
-    delete customProps.closePortal;
-    const platformIsiOS = !!navigator.platform && /iPad|iPhone|iPod/.test(navigator.platform);
-
-    this.modalContentRef = React.createRef();
+  if (zIndexes.indexOf(zIndex) >= 0) {
+    zIndexLayer = zIndex;
+  }
 
-    if (fallbackFocus) {
-      this.fallbackFocus = fallbackFocus;
-    } else {
-      this.fallbackFocus = () => (this.modalContentRef.current);
-    }
+  const modalClassName = cx(['abstract-modal', {
+    'is-fullscreen': isFullscreen,
+  }, `layer-${zIndexLayer}`, classNameModal]); // Delete the closePortal prop that comes from react-portal.
 
-    return (
-      <FocusTrap
-        paused={!isFocused}
-        focusTrapOptions={{
-          escapeDeactivates: false,
-          fallbackFocus: this.fallbackFocus,
-        }}
+  delete customProps.closePortal;
+  const platformIsiOS = !!navigator.platform && /iPad|iPhone|iPod/.test(navigator.platform);
+  return (
+    <React.Fragment>
+      <ModalOverlay
+        onClick={closeOnOutsideClick ? onRequestClose : null}
+        className={classNameOverlay}
+        zIndex={zIndex}
+      />
+      {
+        /*
+          When an aria-label is set and tabIndex is set to 0, VoiceOver will read
+          the aria-label value when the modal is opened
+        */
+        /* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+      }
+      <div
+        tabIndex={platformIsiOS ? '-1' : '0'}
+        aria-label={ariaLabel}
+        className={modalClassName}
+        role={role}
+        ref={(node) => { props.refCallback(node); }}
+        {...customProps}
```

Comment:
want to move customProps as the first attribute

Reply:
Updated in this commit: https://github.com/cerner/terra-framework/pull/608/commits/5ee4e0544b820356cb994d4c507499efa1fa9be7